gameFinished.test.js

```diff
@@ -0,0 +1,27 @@
+const Game = require("./game");
+
+describe(Game.Game, () => {
+  it("throws error if user attempts to input more scores after a perfect game has ended", () => {
+    const game = new Game.Game();
```

Comment:
slight typo here I think? 'Game.Game'

Reply:
This is on purpose due to how both the classes are exported. To access game: Game.Game. To access Frame: Game.Frame